components/EditOnGithub.js

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from "next/router";
+import { RiCodeSSlashFill, RiEdit2Fill, RiFeedbackLine } from "react-icons/ri";
+
+const className =
+  "hover:bg-primary-low group hover:text-secondary-high dark:hover:text-secondary-high flex w-full text-left rounded-md p-2 gap-x-3 text-sm leading-6 font-semibold text-primary-high dark:text-primary-low-medium";
+
+const iconClass = "h-5 w-5 self-center";
```

Comment:
I think these constants should go in the function below also

Reply:
Should these changes be made in a new PR or is it okay to make the changes in the PR which fix the issue #8046

Reply:
That is a separate area of the code, I would make a separate PR for these changes